docs/sources/enterprise/license-restrictions.md

```diff
@@ -23,9 +27,13 @@ In the context of licensing, each user is classified as either a viewer or an ed
 
 Restrictions are applied separately for viewers and editors.
 
-When the number of maximum active viewers or editors is reached, Grafana displays a warning banner.
+When the number of maximum active viewers or editors is reached, only them can log in. New users or non-active users can't log in.
+
+### Concurrent sessions limit
+
+Sometimes it is useful to log in to an account from multiple locations at the same time. With Grafana Enterprise 7.5+, accounts are limited to the number of concurrent sessions authorized in your license (default to three). A new session is created when a user logs in to Grafana from a new device, a different browser, or an incognito window, but not when a user logs in to Grafana from another tab or window within the same browser. Grafana can be open in many browser tabs while still using only one session.
```

Comment:
Regarding 

> accounts are limited to the number of concurrent sessions authorized in your license (default to three)

I don't want to make it obvious to customers that we can customize their concurrent session limit since I think many customers will pressure us to raise/remove their limit, and we don't want to do that. I want to just say something like "accounts are limited to three concurrent sessions" and leave it at that. However, some (very few) customers will have different limits and could be confused if we do that.

@osg-grafana can you think of a creative way to handle this?

Reply:
@osg-grafana WDYT about Mitch's suggestion? Should I replace `accounts are limited to the number of concurrent sessions authorized in your license (default to three)` with `accounts are limited to three concurrent sessions`?

Reply:
Sometimes it is useful to sign in to an account from multiple locations simultaneously. As of Grafana Enterprise 7.5+, accounts are limited to the number of concurrent sessions authorized in your license, which is typically three. A new session is created when you sign in to Grafana from a new device, a different browser, or an incognito window. If a signs in to Grafana from another tab or window within the same browser, then only one licensed session is used.

Reply:
(I think that my feedback got hidden somehow.) Here it is ^^ in a comment instead.